Unix/base/messages.h

```diff
@@ -1395,6 +1397,27 @@ MI_INLINE HttpResponseMsg* __HttpResponseMsg_New(
     return msg;
 }
 
+MI_INLINE HttpResponseMsg* __HttpResponseMsgWithAuth_New(
+    _In_ Page * page,
+    int httpErrorCode,
+    _In_ const char *const authResponseType,
```

Comment:
I'm using the v2 version, but I assume as part of the api it would be maintained. 
If its not needed by anyone else I have no issue with getting rid of it, or folding the other version into that name.